### PR TITLE
Gameplay

### PR DESCRIPTION
**Pull request by sevresbabylone**
- Revises database schema
image

Play.vue: 
- Set a realtime observer to the user's current game. If no current game exists, new game is created and assigned

Creating a game
- Each game is assigned an array of sampleIds based on shuffling method specified in config file. Least seen and Random.
image

Widget.vue:
- Displays correct widget based on config
- Displays screens and handles game's current state (In Progress, Paused, Game over, )
- Allows user to restart game if game is over
- Switches to new card
- Updates sample (totalSeenCount, average)
- Sends vote to votes table
- Updates userSeenSamples table

WordSwipe.vue
- Evaluates if a vote is correct or wrong

Card.vue
Card for each sample that can be sent left or right

### CODE REVIEW

File: src/example.wordswipe-test.config.js [hunk 1]
```diff
@@ -5,14 +5,21 @@ export default {
...
     tagline: "Are the words real or fake?",
   },
-  needsTutorial: true,
+  // shufflingMethod: "random",
+  shufflingMethod: "leastSeen",
+  needsTutorial: false,
   manifestUrl: "",
-  widgetType: "TimedWordSwipe",
+  widgetType: "WordSwipe",
+  mode: "Assessment", // "Data Annotation"
```
Comment (kylesome) [inline src/example.wordswipe-test.config.js:21]: Some comments might be good for new config values explaining the possible values they can take.

File: src/example.wordswipe-test.config.js [hunk 2]
```diff
@@ -5,14 +5,21 @@ export default {
...
+    appId: "1:934860809801:web:b59be557ac5a62105a4198",
+    measurementId: "G-0MCHLFQVZ9",
   },
   home: {
     title: "Rapid Online Assessment of Reading",
     tagline: "Are the words real or fake?",
   },
-  needsTutorial: true,
+  // shufflingMethod: "random",
+  shufflingMethod: "leastSeen",
```
Comment (kylesome) [inline src/example.wordswipe-test.config.js:17]: Some comments might be good for new config values explaining the possible values they can take.

**Review by kylesome — COMMENTED** [review]
Looks good 👍 
Just a small question about the config.
Also saw an empty file [here](https://github.com/SwipesForScience/SwipesForScience/pull/141/files#diff-64afe2288cb987bf59d20b141715cb1b818ff2466e2922105a94cf7c455c2243) still required?

File: src/example.wordswipe-test.config.js (hunk 2 above)
Comment (sevresbabylone) [reply src/example.wordswipe-test.config.js:20]: added some comments, will definitely need to document this in a better way. thanks!

**Pull request merged**